lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -28,6 +28,8 @@ import java.nio.file.Path
 import java.nio.file.Paths
 import kotlin.io.path.readText
 
+private const val ACCOUNT_NAME = "standard_account_test"
```

Comment:
Is there any reason we can't use the default account name `__default__`? If I understand correctly, we could remove `ACCOUNT_NAME` and its usages from all the files, and things should work as expected with the default account name.

Reply:
We can, I wanted to be consistent with previous approach, which assumed passing account names instead of using the default name.
e.g. of previous implementation in `ProviderTest`
https://github.com/software-mansion/starknet-jvm/blob/0ec0897cf9f85ae430d2d56f44c8d8b6dc7be18f/lib/src/test/kotlin/starknet/provider/ProviderTest.kt#L48

Reply:
It seems to me, based on the example you provided, that this deployment is used only to obtain the transaction hash of the `DeployAccount` transaction, which is later used in some tests. The execution of transactions (for example by `InvokeContract`) is still performed using the default account. We can confirm with @DelevoXDG to see if there were any other reasons for this approach. However, I believe we should do everything with the default account to simplify the code, as I don't see significant value in introducing a different account name for every individual test file.